cilium/cmd/config.go

```diff
@@ -42,7 +43,8 @@ var configCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(configCmd)
 	configCmd.Flags().BoolVarP(&listOptions, "list-options", "", false, "List available options")
-	configCmd.Flags().BoolVarP(&listReadOnlyConfigurations, "read-only", "r", false, "Display read only configurations")
+	configCmd.Flags().BoolVarP(&listReadOnlyConfigurations, "read-only", "r", false, "Display read only Configurations")
+	configCmd.Flags().BoolVarP(&listAllConfigurations, "all", "a", false, "Display all cilium Configurations")
```

Comment:
a super minor nit, but configurations probably doesn't need to be capitalized:
```suggestion
	configCmd.Flags().BoolVarP(&listReadOnlyConfigurations, "read-only", "r", false, "Display read only configurations")
	configCmd.Flags().BoolVarP(&listAllConfigurations, "all", "a", false, "Display all cilium configurations")
```